kinds/preferences/preferences_kind.cue

```diff
@@ -0,0 +1,36 @@
+package kind
+
+name: "Preferences"
+maturity: "merged"
+
+lineage: seqs: [
+	{
+		schemas: [
+			{//0.0
+				// UID for the home dashboard
+				homeDashboardUID?: string
+
+				// The timezone selection
+				// TODO: this should use the timezone defined in common
+				timezone?: string
```

Comment:
NOTE: this would be better if we could reference `common.TimeZone` -- but lets solve that independently:
https://github.com/grafana/grafana/pull/61725